satellite/marketing/marketingweb/server_test.go

```diff
@@ -0,0 +1,86 @@
+package marketingweb
+
+import (
+	"net/http"
+	"testing"
+	"net/url"
+	"strings"
+	"log"
+)
+
+
+func buildForm(isReferralOffer bool) url.Values{
+	form := url.Values{}
+	form.Add("Name","May Credit")
+	form.Add("Description","desc")
+	form.Add("ExpiresAt","2019-06-27")
+	form.Add("InviteeCreditInCents","50")
+	form.Add("InviteeCreditDurationDays","50")
+	form.Add("RedeemableCap","150")
+
+	if isReferralOffer {
+		form.Add("AwardCreditInCents","50")
+		form.Add("AwardCreditDurationDays","50")
+	}
+	return form
+}
+
+func buildResources(endpoint string, isReferralOffer bool) (url.URL, url.Values){
+	URL, err := url.ParseRequestURI("http://127.0.0.1:10003")
+	if err != nil{
+		log.Fatalf("URL parsing Err : %v\n", err)
```

Comment:
I don't think we want to exit the process if there is a error here

Reply:
How come? Should I just log the error and return it ?

Reply:
Because we also want other tests continue to run. If you check other tests files, you will see we are using the `"github.com/stretchr/testify/require"` assertion package to do testing

Reply:
pushed